src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -351,10 +351,44 @@ object TezosTypes {
     counter: Int
   )
 
-  final case class BlockAccounts(
-      blockHash: BlockHash,
-      blockLevel: Int,
-      accounts: Map[AccountId, Account] = Map.empty
+  final case class BlockTagged[T](
+    blockHash: BlockHash,
+    blockLevel: Int,
+    content: T
+  ) {
+    val asTuple = (blockHash, blockLevel, content)
+  }
+
+  final case class Delegate(
+    balance: PositiveBigNumber,
+    frozen_balance: PositiveBigNumber,
```

Comment:
This file has a mixed style of writing keys (sometimes underscore, sometimes snakeCase)

Reply:
I noticed too, it's a legacy leftover from the fact that the first types where roughly matching the json-schema from tezos, while later on I introduced an exact match, because I wanted to reduce any ambiguity on the mapping.

What would you suggest? We might open a new PR, but we need to decide if we prefer an exact attribute naming match or only a rough camelCase-converted match

Reply:
I prefer camelCase if that not makes anything significantly harder.

Reply:
@vishakh @ba205 @piotrkosecki can I have your vote in, too?

Reply:
I've opened issue #368 to make the eventual changes